core/src/main/resources/hudson/model/AllView/noJob.groovy

```diff
@@ -111,7 +111,7 @@ div {
 
             h1(_("Welcome to Jenkins!"))
 
-            p("Log in now to view or create jobs. If you don’t already have an account, you can sign up.")
+            p("Log in now to view or create jobs. If you don't already have an account, you can sign up.")
```

Comment:
```suggestion
            p(_("blurb"))
```

Should be made localizable anyway.

Reply:
Out of scope :)

Reply:
Touching a line and not fixing the obvious bug on it is 👎